base/applications/osk/settings.c

```diff
@@ -126,35 +222,49 @@ VOID LoadSettings(VOID)
     Globals.PosX = CW_USEDEFAULT;
     Globals.PosY = CW_USEDEFAULT;
 
+    /* Set font value defaults */
+    Globals.FontHeight = 15;
```

Comment:
Is it possible to assign meaningful macro names for such values rather than hardcoding them?

Reply:
I can create a macro or just leave it as 0 and let the GDI do as it wills, what would be the preferred approach?

Reply:
I'm more comfortable if you create a macro specifically for this.